pkg/destroy/gcp/iam.go

```diff
@@ -39,21 +39,23 @@ func (o *ClusterUninstaller) deleteServiceAccount(serviceAccount string) error {
 	if err != nil && !isNoOp(err) {
 		return errors.Wrapf(err, "failed to delete service account %s", serviceAccount)
 	}
+	o.Logger.Infof("Deleted service account %s", serviceAccount)
 	return nil
 }
 
 // destroyServiceAccounts removes service accounts with a display name that starts
 // with the cluster's infra ID.
-func (o *ClusterUninstaller) destroyServiceAccounts() error {
+func (o *ClusterUninstaller) destroyServiceAccounts() (bool, []error) {
```

Comment:
https://godoc.org/k8s.io/apimachinery/pkg/util/errors#Aggregate
should be better than returning list of errors..?

the list is only useful, if the caller can take action based on type of one of the errors...

Reply:
ack ... will look into leaving the function signature as before